example/emacs.yml

```diff
@@ -32,6 +34,8 @@ keymap:
       M-d: [C-delete, { set_mark: false }]
       # Kill line
       C-k: [Shift-end, C-x, { set_mark: false }]
+      # Kill word backward
+      Alt-backspace: [C-backspace, C-x, {set_mark: false}]
```

Comment:
In my environment (Ubuntu 22.04, X11 GNOME), `C-backspace` already deletes a word, so `C-x` seems to do nothing. Why did you add `C-x` here?

Reply:
Sorry, my fault, i modified config from the copy of above `C-k:`  :smile:

I added a C-M-Space config too, please check.